packages/terra-consumer-nav/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 ChangeLog
+# 0.2.11 - (December 18, 2017)
+### Changed
+- Updated the hover background for help-button and popup content to make it
+- consistent with profile button and content.
```

Comment:
don't split updates, makes the list a little strange to read

Reply:
+1